airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -212,7 +212,8 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
                 'airflow-worker': worker_uuid,
                 'dag_id': dag_id,
                 'task_id': task_id,
-                'execution_date': execution_date
+                'execution_date': execution_date,
+                'try_number': str(try_number)
```

Comment:
The redundant comma thing would be nice here to my mind.
```suggestion
                'try_number': str(try_number),
```